Add `draft2020` to supported schema versions in `help`

jamietanna opened the pull request:
As `draft2020` is supported, we should make it clear in the output from
our `help` subcommand.

jamietanna committed:
Add `draft2020` to supported schema versions in `help`

As `draft2020` is supported, we should make it clear in the output from
our `help` subcommand.

jamietanna commented:
I'm not sure the errors presented are related to the change?

aardvarkk commented:
@jamietanna It looks like they also support `jtd` as a spec -- might this be a good PR to add that in at the same time?

jamietanna committed:
Add `jtd` to supported schema versions in `help`

As `jtd` is supported, we should make it clear in the output from our
`help` subcommand.

jamietanna commented:
Good thinking @aardvarkk, done :+1:
